schema_generator/utils.py

```diff
@@ -0,0 +1,16 @@
+def merge_schemas(a, b):
+    if not (isinstance(a, dict) and isinstance(b, dict)):
+        raise ValueError("Error merging '{}' and '{}'".format(
```

Comment:
This should probably be a TypeError, and the error message could probably be more descriptive: "Unable to merge disparate types: '{}' and '{}'."